app/Controllers/StaticPagesController.js

```diff
@@ -0,0 +1,47 @@
+const catchAsync = require('../utils/catchAsync');
+const path = require('path');
+const ApiError = require('../utils/ApiError.js');
+const httpStatus = require('http-status');
+
+const renderHandler = catchAsync(async (res, page) => {
+  const options = { root: path.join(__dirname, '../../') };
```

Comment:
what about making it relative to `process.cwd()` ? I used that a while ago also for the db path. So the files/folders are relative to the current folder the `vocascan-server web` cmd was executed.